test/common/upstream/load_balancer_fuzz_base.cc

```diff
@@ -0,0 +1,118 @@
+#include "test/common/upstream/load_balancer_fuzz_base.h"
+
+#include "test/common/upstream/utility.h"
+
+namespace Envoy {
+namespace Upstream {
+
+void LoadBalancerFuzzBase::initializeASingleHostSet(uint32_t num_hosts_in_host_set,
```

Comment:
It's a TODO to initialize healthy/degraded hosts too and split into localities, right? feel free to resolve if so

Reply:
Yeah, but I don't think we should initialize explicit healthy/degraded hosts. That functionality is taken care of by updateHealthFlagsForHostSet. Moving all that logic up here would clutter things up. Yes, splitting into localities is what I'm working on today.